tests/hazmat/primitives/test_asym_utils.py

```diff
@@ -61,20 +61,14 @@ def test_encode_dss_non_integer():
         encode_dss_signature("hello", "world")
 
 
-def test_decode_dss_trailing_bytes():
-    with pytest.raises(ValueError):
-        decode_dss_signature(b"0\x06\x02\x01\x01\x02\x01\x01\x00\x00\x00")
```

Comment:
As I mentioned in OP because a number of protocols actually use trailing bytes (which are not reflected by preceding length indicators).